apps/prairielearn/src/components/EditQuestionPointsScore.html.ts

```diff
@@ -37,6 +37,7 @@ export function EditQuestionPointsScoreButton({
     data-html="true"
     data-placement="auto"
     title="Change question ${label}"
+    aria-label="Edit question ${label}"
```

Comment:
Consider using "Change" as the verb here, even though the component has "Edit" in the name. This will match the popover title.

Reply:
Updated to "Change question" in [3b5665b](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/3b5665b6bc0be4706a91dfd033bf10fb8b256e2e).